vectortile/src/main/scala/geotrellis/vectortile/Layer.scala

```diff
@@ -215,13 +215,15 @@ ${sortedMeta.map({ case (k,v) => s"            ${k}: ${v}"}).mkString("\n")}
   tileWidth: Int,
   version: Int,
   tileExtent: Extent,
-  points: Seq[MVTFeature[Point]],
-  multiPoints: Seq[MVTFeature[MultiPoint]],
-  lines: Seq[MVTFeature[LineString]],
-  multiLines: Seq[MVTFeature[MultiLineString]],
-  polygons: Seq[MVTFeature[Polygon]],
-  multiPolygons: Seq[MVTFeature[MultiPolygon]]
-) extends Layer
+  mvtFeatures: MVTFeatures
```

Comment:
There needs to be a constructor overload to go from old list of args to mvtFeatures to keep this PR from being binary incompatible with previous version. Otherwise this is a solid improvement.

Reply:
9e10641